### ISSUE TITLE
PDF Output?

### ISSUE THREAD

**Issue by willzfarmer** [jupyter-incubator/dashboards#260]
Hey, this is awesome! Do you have any plans to make these dashboards exportable as PDFs or just raw HTML? Being able to format cells like this is much nicer for some stuff than the normal notebook...

**Comment by dalogsdon** [comment]
We have noticed the potential that the dashboard layout has but to my knowledge haven't discussed static outputs such as PDF or raw HTML. You say it would be nicer for some stuff; do you have a specific use case in mind?

**Comment by willzfarmer** [comment]
Yeah, so for instance, if I wanted to arrange different cells to be side by side, currently the best-ish way to do it is to either save the output images and re-render with markdown/display use the display module, or use matplotlib to show images inline. This doesn't work as well for something like tables however.

Other examples include being able to (hypothetically) create a static website using a series of these HTML-output dashboards.

Being able to print dashboard views without having to screenshot them is a plus.

etc.

The big thing I would use it for is more control over how documents that I write in the notebook look. Currently I use the standard `nbconvert --to pdf <notebook>` method to get a pdf, but it sometimes has issues with image size, etc. Having a more fine-grained level of control would be nice.

**Comment by parente** [comment]
If the metadata spec for the layouts becomes something standard across the various jupyter tools, then nb convert could perhaps respect the layout when converting to various formats.

The spec is currently on the wiki here. (Sorry, on mobile, else I'd link)

**Comment by jtyberg** [comment]
https://github.com/jupyter-incubator/dashboards/wiki/Dashboard-Metadata-and-Rendering

**Comment by willzfarmer** [comment]
Also, another good reason to support PDF output is sometimes it's nice to be able to reorganize cells dynamically and not have to have all the code inline. I generate a lot of reports using the notebook and I've had to use different tools because there's not currently a way to suppress code input cells. This dashboard view supports that however (notably the report view).
